src/main/scala/io/dronekit/cloud/AWSWrapper.scala

```diff
@@ -170,7 +170,7 @@ class AWSWrapper(S3Client: AmazonS3Client = S3.client)
   }
 
   def streamUpload(s3url: S3URL): Sink[ByteString, Future[CompleteMultipartUploadResult]] = {
-    multipartUploadTransform(s3url).fold(Seq.empty[UploadPartResult])(_ :+ _).to(Sink.head)
+    multipartUploadTransform(s3url).fold(Seq.empty[UploadPartResult])(_ :+ _).to(Sink.ignore)
```

Comment:
Is there a reason you kept the `fold`? Ignoring each UploadPartResult should be the same as building a Seq of them and ignoring that.